addons/website_sale_slides/controllers/__init__.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from . import main
```

Comment:
We already have an override of slides controllers just below ^^ (I think it came with the rebase on top of the other slides promenade)

Reply:
ah maybe, didn't notice, good catch :)